app/src/main/java/com/example/concordia_campus_guide/Models/Direction.java

```diff
@@ -2,21 +2,28 @@
 
 import com.example.concordia_campus_guide.Models.Routes.TransportType;
 import com.google.android.gms.maps.model.LatLng;
+
+import java.io.Serializable;
 import java.util.Date;
 
-public class Direction {
-    private LatLng start;
-    private LatLng end;
+public class Direction implements Serializable {
```

Comment:
why does Direction needs to implement serializable ?

Reply:
It was breaking our app because to pass it through activities we were using putSerializable